linkerd/app/src/env.rs

```diff
@@ -441,46 +516,39 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
             keepalive,
             h2_settings,
         };
-        let cache_max_idle_age =
-            outbound_cache_max_idle_age?.unwrap_or(DEFAULT_OUTBOUND_ROUTER_MAX_IDLE_AGE);
-        let max_idle =
-            outbound_max_idle_per_endpoint?.unwrap_or(DEFAULT_OUTBOUND_MAX_IDLE_CONNS_PER_ENDPOINT);
-        let keepalive = Keepalive(outbound_connect_keepalive?);
-        let connect = ConnectConfig {
-            keepalive,
-            timeout: outbound_connect_timeout?.unwrap_or(DEFAULT_OUTBOUND_CONNECT_TIMEOUT),
-            backoff: parse_backoff(
-                strings,
-                OUTBOUND_CONNECT_BASE,
-                DEFAULT_OUTBOUND_CONNECT_BACKOFF,
-            )?,
-            h2_settings,
-            h1_settings: h1::PoolSettings {
-                max_idle,
-                idle_timeout: cache_max_idle_age,
-            },
-        };
 
         let detect_protocol_timeout =
             outbound_detect_timeout?.unwrap_or(DEFAULT_OUTBOUND_DETECT_TIMEOUT);
         let dispatch_timeout =
             outbound_dispatch_timeout?.unwrap_or(DEFAULT_OUTBOUND_DISPATCH_TIMEOUT);
-
+        let policy = match policy_svc.clone() {
+            Some(control) => outbound::policy::Config {
+                workload: workload.clone(),
+                control,
+            },
+            None => {
+                // TODO(eliza): should we just disable client policy if there's no
+                // policy service address?
+                error!("{}_ADDR must be set", ENV_POLICY_SVC_BASE);
+                return Err(EnvError::InvalidEnvVar);
+            }
```

Comment:
should we require that a policy service address be provided, or just disable client policy discovery if there's no policy service addr?